src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/AmendApplicationAboutToSubmitHandlerTest.java

```diff
@@ -83,7 +89,7 @@ public void given_case_when_wrong_casetype_then_case_can_not_handle() {
     @Test
     public void given_case_when_wrong_eventtype_then_case_can_not_handle() {
         assertThat(handler
-                .canHandle(CallbackType.ABOUT_TO_SUBMIT, CaseType.CONSENTED, EventType.CLOSE),
+                .canHandle(CallbackType.ABOUT_TO_SUBMIT, CONSENTED, EventType.CLOSE),
```

Comment:
same as above

Reply:
you can delete this method since the test has been included by`givenCase_whenEventIsAmendApplication_thenCanHandle`

Reply:
can you delete test `given_case_when_wrong_casetype_then_case_can_not_handle` as well please?

Reply:
[0a371f0](https://github.com/hmcts/finrem-case-orchestration-service/pull/1753/commits/0a371f0ae27cc77534bd50ff33aaef8474616e29)